elements/demos/src/SingleSelectionDemo.js

```diff
@@ -0,0 +1,46 @@
+import SingleSelectionMixin from '../../elix-mixins/src/SingleSelectionMixin';
+import symbols from '../../elix-mixins/src/symbols';
```

Comment:
Great question. The symbols effectively work as named channels for communication between mixins or between a mixin and the component class being defined. So if you're creating a component, you'll need to import `symbols` if and only if:
* your component class needs to invoke a method or property defined by a mixin.
* your component class needs to supply a method or property that a mixin is expecting.

Our mixin documentation should let you know which symbols they use. E.g., the [initial documentation](https://github.com/JanMiksovsky/elix/blob/list-mixins-1/elements/elix-mixins/docs/SelectionAriaMixin.md) for [SelectionAriaMixin](https://github.com/JanMiksovsky/elix/blob/list-mixins-1/elements/elix-mixins/src/SelectionAriaMixin.js) indicates that the mixin expects the following to be defined: `symbols.itemSelected`, `symbols.itemAdded`, and "selectedItem" (a string name, because it's part of the component's public API). Those members can be defined by the component itself, or supplied by other mixins.

Ultimately, the `symbols` file will list all the properties and methods for which the project has defined standard semantics. As you note, most components using Elix mixins will need to import that file. That's a benefit — the use of `Symbol` objects to identify those properties and methods ensures that a developer can't accidentally give a property/method a name that collides with one of ours. They have to explicitly opt in to using our list of known symbols by importing that file.

In some cases you'll be able to compose complete, functioning components by applying a collection of mixins, and the component class won't need to provide or invoke any `Symbol`-identified properties or methods. In those cases, you wouldn't import `symbols`.